examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -32,6 +32,7 @@
 from accelerate import Accelerator
 from accelerate.logging import get_logger
 from accelerate.utils import DistributedDataParallelKwargs, ProjectConfiguration, set_seed
+from datasets import load_dataset
```

Comment:
Actually we don't have a requirement on `datasets` and I'd like to keep it this way

Reply:
We do have it in `examples/controlnet/train_controlnet.py` though: https://github.com/huggingface/diffusers/blob/a042909c836c794a34508b314b3ce8ce93a96284/examples/controlnet/train_controlnet.py#L33C34-L33C34